src/styles/theming/sap_fiori_3.scss

```diff
@@ -290,6 +290,7 @@
 
   /* Toolbar */
   --fdToolbar_Auto_Background: inherit;
+  --fdToolbar_Solid_Background: var(--sapBackgroundColor);
```

Comment:
Why do we need a variable for this if it's the same value for each theme?

Reply:
nice catch, thank you! I updated the value for Horizon. It should be `--sapToolbar_Background` for Horizon and `--sapBackgroundColor` for Quartz in Solid design.